doc/policies/admin.rst

```diff
@@ -391,12 +391,98 @@ type: bool
 To be able to define new :ref:`smtpserver` or delete existing ones, the
 administrator needs this rights ``smtpserver_write``.
 
+smtpserver_read
+~~~~~~~~~~~~~~~
+
+type: bool
+
+Allow the administrator to read the smtpserver configuration.
+
+smsgateway_write
+~~~~~~~~~~~~~~~~
+
+type: bool
+
+To be able to define new :ref:`smsgateway` or delete existing ones, the
+administrator needs this rights ``smsgateway_write``.
+
+smsgateway_read
+~~~~~~~~~~~~~~~
+
+type: bool
+
+Allow the administrator to read the smsgateway configuration.
+
+periodictask_write
+~~~~~~~~~~~~~~~~
+
+type: bool
+
+Allow the administrator to write or delete periodictask definitions.
+
+periodictask_read
+~~~~~~~~~~~~~~~
+
+type: bool
+
+Allow the administrator to read the periodictask definitions.
+
 eventhandling_write
 ~~~~~~~~~~~~~~~~~~~
 
 type: bool
 
-Allow the admiinstrator to configure :ref:`eventhandler`.
+Allow the adminstrator to configure :ref:`eventhandler`.
+
+eventhandling_read
+~~~~~~~~~~~~~~~~~~
+
+type: bool
+
+Allow the administrator to read event handlers.
+
+.. note:: Currently the policies do not take into account resolvers,
+   or realms. Having the right to read event handlers, will allow the
+   administrator to see all policies.
```

Comment:
I think this should be *to see all event handlers?

Reply:
right. Will do.

Reply:
done in 003f1a44
(might be a good idea to resolve conversations)